src/EditorFeatures/VisualBasicTest/CodeActions/ExtractMethod/ExtractMethodTests.vb

```diff
@@ -293,5 +293,21 @@ End Class
 </Text>.Value.Replace(vbLf, vbCrLf),
 compareTokens:=False)
         End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractMethod), Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.Tuples)>
+        <WorkItem(13042, "https://github.com/dotnet/roslyn/issues/13042")>
+        Public Async Function TestTuples() As Task
+
+            Await TestAsync(
+NewLines("Class Program \n Sub Main(args As String()) \n [|Dim x = (1, 2)|] \n M(x) \n End Sub
```

Comment:
you don't need this either.  VB natively supports multiline string literals now :)  SO just make a single string literal that contains the test you want.